invoices/invoiceregistry.go

```diff
@@ -198,12 +198,7 @@ func (i *InvoiceRegistry) populateExpiryWatcher() error {
 		return err
 	}
 
-	for idx := range pendingInvoices {
-		i.expiryWatcher.AddInvoice(
-			pendingInvoices[idx].PaymentHash, &pendingInvoices[idx].Invoice,
-		)
-	}
-
+	i.expiryWatcher.AddInvoices(pendingInvoices)
```

Comment:
May be useful to have log statements on either side of this line so we can track any long lived progress in the future.

Reply:
Done.